src/buildstream/utils.py

```diff
@@ -575,20 +576,27 @@ def link_files(
     return result
 
 
-def get_host_tool(name: str) -> str:
+def get_host_tool(
+    name: str,
+    search_subprojects_dir: Optional[str] = None,
```

Comment:
Let's not expose this in public API.

Better to have a `_get_host_tool_internal()` with the added option for internal use and keep public facing `get_host_tool()` as is.

Reply:
Great point! Done

Reply:
I suppose the `search_subprojects_dir` argument (along with the line in the docstring below) should be removed now